cfpb/cfgov-refresh
# Update `fax` to `is_fax` boolean label

## Changes

- Changes `fax` boolean label to `is_fax`, per TODO comment.

## Testing

1. Run `python cfgov/manage.py migrate`.
2. Create a browse page in Wagtail.
3. Add a expandable.
4. Add a Phone block to the expandable.
5. Check that the `Is this number a fax?` checkbox works.

## Screenshots

<img width="998" alt="screen shot 2017-08-04 at 4 38 02 pm" src="https://user-images.githubusercontent.com/704760/28986347-9079cb98-7934-11e7-98eb-f03d378aeb9a.png">


## Notes

- Seems like is_fax should be attached to EACH number, not ALL numbers!
Opened https://github.com/cfpb/cfgov-refresh/issues/3213
